widgetssdk/src/main/java/com/glia/widgets/core/secureconversations/domain/MarkMessagesReadUseCase.kt

```diff
@@ -1,15 +0,0 @@
-package com.glia.widgets.core.secureconversations.domain
-
-import com.glia.androidsdk.RequestCallback
-import com.glia.widgets.core.secureconversations.SecureConversationsRepository
-
-class MarkMessagesReadUseCase(
-    private val secureConversationsRepository: SecureConversationsRepository
-) {
-
-    fun execute(
-        callback: RequestCallback<Void>
-    ) {
-        secureConversationsRepository.markMessagesRead(callback)
```

Comment:
I see `secureConversationsRepository.markMessagesRead(callback)` removed, but cannot find the replacement for this call. I expected to see that we are still calling it somewhere. Don't we need to mark messages as seen?

Reply:
It'll be implemented on next task(I'm currently working on it)